src/main/java/com/Acrobot/ChestShop/Configuration/Properties.java

```diff
@@ -337,4 +337,10 @@ public <T> Object parseToJava(Class<T> type, Object object) {
     @PrecededBySpace
     @ConfigurationComment("Add stock counter to quantity line?")
     public static boolean USE_STOCK_COUNTER = false;
+
+    @PrecededBySpace
+    @ConfigurationComment("Exclude these enchantments from the similarity check when comparing items")
```

Comment:
This comment seems wrong to what it now does. Also might want to add an example of what keys are supported here or a link to a site listing those. (Or at least that info in the PR so it can be added to the wiki)